myblog/blog/models.py

```diff
@@ -1,11 +1,26 @@
 from django.db import models
 from django.contrib.auth.models import User
+from django.utils import timezone
+
+def post_image_upload_path(instance, filename):
+    return 'uploads/images/{}/{}/{}/{}'.format(
+        timezone.now().year, 
+        timezone.now().month, 
+        timezone.now().day, 
+        filename
+    )
```

Comment:
This is not necessary, as ImageField (or more specifically, its parent FileField) supports [strftime formatting.](https://docs.djangoproject.com/en/4.1/ref/models/fields/#filefield)